tensorboard/pip_package/build_pip_package.sh

```diff
@@ -23,60 +23,40 @@ else
   sedi="sed -i"
 fi
 
-run_smoke_test=1
+tf_version="tf-nightly"
+if [ -n "$TF_VERSION" ]; then
```

Comment:
nit: `${TF_VERSION}` for consistency and per style:
<https://google.github.io/styleguide/shell.xml>

Reply:
Done.